core/backend/src/IModelDb.ts

```diff
@@ -365,7 +365,7 @@ export abstract class IModelDb extends IModel {
   }
 
   /** Close this IModel, if it is currently open. */
```

Comment:
please add a comment that this function automatically saves changes.

Reply:
Updated comment to reflect the current behaviour.